modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -1239,6 +1239,10 @@ Included when xref:configuration/configuration-settings.adoc#config_db.logs.quer
 | Reason for failure.
 Included when applicable.
 
+| errorInfo
```

Comment:
Should we somehow specify that this field in only available starting from version 5.25?